packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -7,15 +7,20 @@ const { ApplicationError } = require('@strapi/utils').errors;
 const { getDeepPopulate, getDeepPopulateDraftCount } = require('./utils/populate');
 const { getDeepRelationsCount } = require('./utils/count');
 const { sumDraftCounts } = require('./utils/draft');
+const { ALLOWED_WEBHOOK_EVENTS } = require('../constants');
 
 const { hasDraftAndPublish } = strapiUtils.contentTypes;
 const { PUBLISHED_AT_ATTRIBUTE, CREATED_BY_ATTRIBUTE } = strapiUtils.contentTypes.constants;
-const { ENTRY_PUBLISH, ENTRY_UNPUBLISH } = strapiUtils.webhook.webhookEvents;
 
 const omitPublishedAtField = omit(PUBLISHED_AT_ATTRIBUTE);
 
-const emitEvent = async (event, entity, modelUid) => {
-  const modelDef = strapi.getModel(modelUid);
+const emitEvent = async (uid, eventName, entity) => {
+  const event = ALLOWED_WEBHOOK_EVENTS[eventName];
+  if (!event) {
+    throw new ApplicationError(`The webhook event ${eventName} doesn't exist.`);
+  }
```

Comment:
why do we need this?

could we export from constants something like this:
```js
const { ENTRY_PUBLISH, ENTRY_UNPUBLISH } = require('../constants/webhooks');
```

I think we could do something as before, but importing those webhook events from the package constants.

Reply:
No worries that's a good point, I've addressed it in [`c53b7a7` (#16835)](https://github.com/strapi/strapi/pull/16835/commits/c53b7a7ae4999a6afae158d3b68b0633efc3873f)